apps/src/templates/progress/ProgressDetailToggle.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import ToggleGroup from '../ToggleGroup';
+import color from "@cdo/apps/util/color";
+import { setIsSummaryView } from '@cdo/apps/code-studio/progressRedux';
+import { connect } from 'react-redux';
+
+const summaryActive = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QEfBicvUU//cAAAAKtJREFUOMvtlLENwjAURN93kQoWoGEzNggNVPRxl4ZMAJNBkwVIExccTaJYFpLTIBq/8p/vbMlfB79Ckpc0amGU5CO9SfQcoyRvkl7AJrlvMLPtFPxNzzE4oANCNAzTbCbVcwTgSmHG7s/e662ToAIwCOasPex3F4Dbo2+QzrOeDZz8TqKOTYJKoo7OHteGxn5nRmfRrxsEs2UrUn3Ni8tWlK4oXVG64u9d8QEqLgOqdcEqDwAAAABJRU5ErkJggg==";
+const summaryInactive = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QEfBjoS9kvffQAAALhJREFUOMvtlLERglAQRN/9gCJMbMYOzO0AEojIISORCjTXChxr0YQiCFgDAf9gAASOyd/s397+u5nbWfgRjMutBFIg6mstULHf5QBc7wXqMo+fQwtUDiyeiKK+9oaUrPh01DtQ3U/xJqr+PL/4+Y3NjgSMrjg/m1KdUvUHMmjNWXXYbnKA06MpkDItPOCgdxKxLxJEErHXm2iFKwa9M6M27+oGrRmjK6b8ko2B4IqQFSErQlb8NyteNvx30d8zRqcAAAAASUVORK5CYII=";
+const detailActive = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAAWklEQVR4AWP4//9/GxD//E8lADWrDWTw5//UB59p52I4GAULHzxrXXj/6c8F95/+pwYGmQUykwHZUGoazgBi0AKPGjxqMCamXTpe9PBZG01yHgyMgtE6D+5iAJNEO6l4m/U/AAAAAElFTkSuQmCC";
+const detailInactive = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAAXklEQVR4AWNgWLunDYh/AvF/amCoWW1Ag/d+BglQE4PMpJ2L4WAULHzwrHXh/ac/F9x/+p8aGGQWyEwGZEOpaTgDiEELPGrwqMGYmHbpeNHDZ200yXkwMApG6zy4iwHRcmtODj8COgAAAABJRU5ErkJggg==";
```

Comment:
yep. Just stick the png in a file and:

```jsx
import summaryActive from './summaryActive.png';
```

The webpack config controls whether `summaryActive` gets turned into a data url or just a regular asset url based on the size of the file. Can't remember what it is set to now, but you can override it in your import statement if you want to force it in one way or the other.